cmd/ocsp-updater/main.go

```diff
@@ -19,16 +21,18 @@ import (
 	bgrpc "github.com/letsencrypt/boulder/grpc"
 	blog "github.com/letsencrypt/boulder/log"
 	"github.com/letsencrypt/boulder/sa"
-	"github.com/prometheus/client_golang/prometheus"
 )
 
 /*
- * ocspDB is an interface collecting the gorp.DbMap functions that the
- * various parts of OCSPUpdater rely on. Using this adapter shim allows tests to
+ * ocspDB and ocspReadOnlyDB are interfaces collecting the gorp.DbMap functions that
+ * the various parts of OCSPUpdater rely on. Using this adapter shim allows tests to
```

Comment:
I see that we used the `/* ` syntax in the original code, however the `// ` is more idiomatic. Even though these interfaces aren't exported we should stick to one comment per block.

Reply:
Okay.